src/main/scala/viper/gobra/translator/util/Worksheet.sc

```diff
@@ -1,6 +1,20 @@
-import viper.silver.{ast => vpr}
+import TypePatterns.::
```

Comment:
should this file be part of the code base?

Reply:
no, I will remove it